### PR TITLE
New packages, overrides and fixes.

### PR DESCRIPTION
**Pull request by Dudulle**

### CODE REVIEW

File: opengapps-packages.mk [hunk 1]
```diff
@@ -12,22 +12,28 @@ PRODUCT_PACKAGES += GoogleBackupTransport \
                     SetupWizard \
                     Phonesky \
                     GoogleCalendarSyncAdapter
+                    
+ifneq ($(filter $(call get-allowed-api-levels),23),)
```
Comment (Blystad) [inline opengapps-packages.mk]: Shouldn't GoogleTTS override for LP as well?

File: opengapps-packages.mk [hunk 2]
```diff
@@ -12,22 +12,28 @@ PRODUCT_PACKAGES += GoogleBackupTransport \
...
 
 ifneq ($(filter $(TARGET_GAPPS_VARIANT),micro),) # require at least micro
 PRODUCT_PACKAGES += CalendarGooglePrebuilt \
                     PrebuiltExchange3Google \
                     PrebuiltGmail \
-                    GoogleHome \
-                    GoogleTTS
+                    GoogleHome
+                    
+ifeq ($(filter $(call get-allowed-api-levels),23),)
```
Comment (Blystad) [inline opengapps-packages.mk]: and why is there a duplicate here?

File: opengapps-packages.mk (hunk 1 above)
Comment (mfonville) [inline opengapps-packages.mk]: GoogleTTS was before 6.0 optional, since 6.0 it is required for SetupWizard

File: opengapps-packages.mk (hunk 2 above)
Comment (Dudulle) [inline opengapps-packages.mk]: This one adds tts if the API level is below 23 so it is not added twice on Marshmallow.

**Comment by Blystad** [comment]
Fair enough. Thanks for your contribution :+1:

**Pull request merged**